it_training/Faketify/Faketify/Managers/AuthManager.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  AuthManager.swift
+//  Faketify
+//
+//  Created by Nguyen Duc on 8/7/25.
+//
+
+import Foundation
+
+final class AuthManager {
+    static let shared = AuthManager()
+    
+    struct Constants {
+        static let clientID = "e9f87f02cc0a4a6b9cdc2612eadafecf"
+        static let clientSecret = "7829b08bb2a849748a210770370dc090"
+    }
+    private init() {}
+    
+    public var signInURL: URL? {
```

Comment:
```suggestion
   var signInURL: URL? {
```

đang khai báo thừa access modifier, tương tự với các function khác ở dưới